source/code-snippets/indexes/listIndexes.js

```diff
@@ -0,0 +1,27 @@
+const { MongoClient } = require("mongodb");
+
+// Replace the following with your MongoDB deployment's connection
+// string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+import { MongoClient } from "mongodb";
+const database = client.db("<databaseName>");
+const collection = database.collection("<collectionName>");
+
+async function run() {
+  try {
+    // start listIndexes example
+    const result = await collection.listIndexes().toArray();
+    console.log("The indexes in the collection are: ");
```

Comment:
S: Per the style guide, we do not introduce lists with fragment sentences. (also applies to other file)

Perhaps something like this would be better:
```suggestion
    console.log("Existing indexes:\n");
```